core/src/main/java/com/linecorp/armeria/client/UnprocessedRequestException.java

```diff
@@ -29,19 +34,31 @@
 
     private static final long serialVersionUID = 4679512839715213302L;
 
-    private static final UnprocessedRequestException INSTANCE = new UnprocessedRequestException(false);
+    /**
+     * Creates a new instance with the specified {@code message} and {@code cause}.
+     */
+    public UnprocessedRequestException(@Nullable String message, Throwable cause) {
+        super(message, requireNonNull(cause, "cause"));
+    }
 
     /**
-     * Returns an {@link UnprocessedRequestException} which may be a singleton or a new instance, depending on
-     * whether {@linkplain Flags#verboseExceptions() the verbose exception mode} is enabled.
+     * Creates a new instance with the specified {@code cause}.
      */
-    public static UnprocessedRequestException get() {
-        return Flags.verboseExceptions() ? new UnprocessedRequestException() : INSTANCE;
+    public UnprocessedRequestException(Throwable cause) {
+        super(requireNonNull(cause, "cause"));
```

Comment:
How about passing the string representation of cause as a message?

    super(requireNonNull(cause, "cause").toString(), cause);

Reply:
It looks better than the null message. Thanks for all of your nice and gentle feedback. 😁